app/views/conferences/show.html.haml

```diff
@@ -45,6 +45,10 @@
       %section#program
         = render 'schedule_splashpage'
 
+    - if @conference.program.cfps.for_tracks.try(:open?) && @conference.splashpage.include_cfp
```

Comment:
Why? A conference is probably going to have the call for tracks before the call for papers, so that the tracks are available for proposal submissions